styleguide/template.md

```diff
@@ -113,6 +113,12 @@ To link to a header in a Markdown file in the same repo, use relative linking +
 
 - Example: [.NET Community](../docs/welcome.md#community)
 
+The C# language specification and the Visual Basic language specification are included in the .NET docs by including the source from the language repositories. The markdown sources are managed in the [csharplang](../csharplang) and [visual basic](../vblang) repositories.
```

Comment:
Links to the repositories don't work in the preview. Should those be full links:
https://github.com/dotnet/csharplang
https://github.com/dotnet/vblang

Reply:
I just saw this after hitting the merge button.  

I'll make a new PR for this and your earlier comment regarding the format.

Reply:
See #8298